frontends/web/src/routes/device/waiting.tsx

```diff
@@ -27,6 +32,13 @@ export const Waiting = () => {
   const { t } = useTranslation();
   const { isDarkMode } = useDarkmode();
 
+  const keystores = useKeystores();
+  const devices = useSync(getDeviceList, syncDeviceList);
```

Comment:
i was hoping only the keystore check would be enough but somehow the frontend gets the keystore event way too late (looks like the backend/frontend is overloaded with work or handling events during that time, and we have no way to prioritize certain events at the moment), so i added the devices check too, which necessarily comes earlier (already before the unlock). software keystore